framework/src/main/java/org/tron/core/services/jsonrpc/filters/LogFilterWrapper.java

```diff
@@ -51,15 +51,15 @@ public LogFilterWrapper(FilterRequest fr, long currentMaxBlockNum, Wallet wallet
       // then if toBlock < maxBlockNum, set fromBlock = toBlock
       // then if toBlock >= maxBlockNum, set fromBlock = maxBlockNum
       if (StringUtils.isEmpty(fr.getFromBlock()) && StringUtils.isNotEmpty(fr.getToBlock())) {
-        toBlockSrc = JsonRpcApiUtil.getByJsonBlockId(fr.getToBlock());
```

Comment:
The method `public static long getByJsonBlockId(String blockNumOrTag)` is only used in LogFilterWrapper, and it should not support `finalized`. So it should `throw new JsonRpcInvalidParamsException("TAG finalized not supported");`